### DISCUSSION TITLE
How to handle dynamic nested fields

### DISCUSSION THREAD

**Discussion by timmyd87** [josefarias/hotwire_combobox#96] [Q&A] [answered]
Hey team, this is a great gem! 

One question is how to handle and target a combobox where you have dynamic nested fields...can we pass the object somehow in the turbo response to then target the form input in the stream? 

Cheers
Tim

**Comment by josefarias** [comment]
Hey @timmyd87, thanks! Not sure I follow. Can you provide more information on what you're trying to do? Some code and plenty of specifics would be great.

**Reply by timmyd87** [reply]
Sure, sorry for not being clear.

With a package like - https://www.stimulus-components.com/docs/stimulus-rails-nested-form -  you can create nested fields dynamically - it works by inserting a unique number (normally time.now) so differentiate the fields - something like name="user[todos_attributes][1710211207871][description]"

I understand with the turbo response we need to target the field name/id however when it has a dynamic number as the above example, how could we pass this as part of the round turbo trip?

Is that clearer? (thank you for the quick response)

**Reply by timmyd87** [reply]
Ummm @josefarias i think i just answered my own question.

I removed the 'for_id' in the turbo_stream.erb response and it worked with the dynamic fields...is it handling it out of the box? If so, you're simply amazing.

**Reply by josefarias** [reply, marked as answer]
Ha, yeah that was a suggestion from @excid3, actually! Who is in fact amazing. `for_id` is not needed in the response anymore! But it's there if you want to override it.

Closing for now but let me know if it ends up not working again!

**Reply by timmyd87** [reply]
Haha @excid3 is indeed amazing! Great, thanks @josefarias, this is a phenomenal gem, i rolled one of these myself and so glad i can replace it now. Thanks again for your help!
